### ISSUE TITLE
pendraggon's ERROR issue

### ISSUE THREAD

**Issue by pendragg0n** [loot/loot#1146]
So.
Using Wrye Bash, game is Special Edition.
I'm in the middle of creating my new build, I've used I've used the "NPC's run and walk at my pace" mod, at https://www.nexusmods.com/skyrimspecialedition/mods/2482
It has operated well in every play-through with no issues.
So I put in my new build, ran LOOT, and it threw this error:

Error
Oh no, something went wrong! This is usually because "C:\Users\Grandpa\AppData\Local\Skyrim Special Edition\plugins.txt" is set to be read-only. If it is, unset it and try again. If it isn't, you can check your LOOTDebugLog.txt (you can get to it through the main menu) for more information.

LOOTDebugLog.txt:

[11:07:36.686513] [warning]: Blocking load of resource at https://fonts.googleapis.com/css?family=Roboto+Mono:400,700|Roboto:400,300,300italic,400italic,500,500italic,700,700italic
[11:07:50.408298] [error]: libloadorder failed to set the load order. Details: Text could not be encoded in Windows-1252
[11:07:50.408298] [error]: Exception while executing query: libloadorder failed to set the load order. Details: Text could not be encoded in Windows-1252: Libloadorder error
[11:07:50.409298] [error]: Chromium console message from http://loot/ui/app.bundle.js at line 5039: Error: Oh no, something went wrong! This is usually because "C:\Users\Grandpa\AppData\Local\Skyrim Special Edition\plugins.txt" is set to be read-only. If it is, unset it and try again. If it isn't, you can check your LOOTDebugLog.txt (you can get to it through the main menu) for more information.
    at onFailure (http://loot/ui/app.bundle.js:5039:5817) *

.../plugins.txt is Not set to read only, never has been.
I am using the most current version of the "NPC's Run and Walk at Your Pace" mod, even though it is a few years old.
I even ran LOOT independently of Wrye Bash got the same results.

Does anybody have a clue as to what the issue is?

Here's my load order my model list:

https://pastebin.com/hagGkv6W 

* The "<http://loot/ui/app.bundle.js:5039:5817>" link does not go to a page that has anything to do with the LOOT mod.

**Comment by Ortham** [comment]
This is probably because there's a plugin filename that contains characters that can't be represented in the Windows-1252 encoding that the game reads plugins.txt as. The debug log should contain a list of plugins to help identify which one it is.

**Comment by CptMcSplody** [comment]
> Here's my load order my model list:
> 
> https://pastebin.com/hagGkv6W

```
Active Mod Files:
[spoiler][xml]
00  Skyrim.esm
01  Update.esm
02  Dawnguard.esm
03  Hearthfires.esm
04  Dragonborn.esm
05  Unofficial Skyrim Special Edition Patch.esp  [Version 4.1.9]
06  Skyrim Project Optimization - Full Version.esm
07  FISS.esp
08  EFFCore.esm
09  UnreadBooksGlow.esl  [Version 2.21]
0A  OresFirewoodsSettings.esl
0B  SMIM-SE-RiftenDocksRopesObjectFix.esp
0C  SMIM-SE-FurnitureChestSnowFix.esp
0D  SMIM-SE-FarmhouseFlickeringFix.esp
0E  SMIM-SE-SolitudeDocksFixes.esp
0F  SMIM-SE-DungeonsCliffsIceSkirts.esp
10  SMIM-SE-DragonbornTernFix.esp
11  SkyUI_SE.esp
12  RelightingSkyrim_SSE.esp
13  Weapons Armor Clothing & Clutter Fixes.esp
14  Cutting Room Floor.esp  [Version 3.1.5]
15  Book Covers Skyrim.esp
16  WACCF_Armor and Clothing Extension.esp
17  Qw_BookCoversSkyrim_CRF Patch.esp
18  Qw_ACE_CRF Patch.esp
19  Qw_WACCF_CRF Patch.esp
1A  Qw_RelightingSkyrim_CRF Patch.esp
1B  LootableCratesSE.esp
1C  Tish.esp
1D  Nicky.esp
1E  JetteCragJumper.esp
1F  Mae.esp
20  Seraphim - PrettyCourier.esp  [Version 5.0]
21  EFFDialogue.esp
22  Run For Your Lives.esp  [Version 4.0]
23  Immersive Horses.esp
24  AMatterOfTime.esp  [Version 2.00]
25  FNIS.esp
26  FNISspells.esp
27  CBBE.esp
28  RaceMenuMorphsCBBE.esp
29  RaceMenu.esp
2A  XPMSE.esp
2B  Realistic Ragdoll Force - Reduced.esp
2C  icepenguinworldmapclassic.esp
2D  SINful CBP.esp
2E  NQS_NamedQuicksaves.esp
2F  Invisible Headgear-WACFF+ACE - espfe.esp
30  Consistency Patch - WACCF and ACE.esp
31  Artful Dodger - Dynamic Pickpocket Cap.esp
32  PickUpBooksSimple.esp
33  EastmarchOreVeins.esp
34  EzEWorldMapSE.esp
35  Headtracking.esp
36  ThaneWeaponsReborn.esp
37  Realistic-Voice.esp
38  Book Covers Skyrim - Immersive Horses Patch.esp
39  LeftHandRings_1stPersonView.esp
3A  LeftHandRings_1stPersonView+USSEP.esp
3B  No NPC Greetings.esp
3C  PeopleAreStrangers.esp
3D  Don'tTalkWithYourMouthFull.esp
3E  Move it Dammit - Less Wait Time and Follow Me Closer.esp  [Version 1.1]
3F  No Killmoves, No Killcams, No Killbites.esp
40  No More Standing Too Close SSE 1-0.esp
41  ShutUpBrynjolf.esp
42  ChargeDialogueFix.esp
43  Ordinator - Perks of Skyrim.esp
44  Ordinator_WACCF_Patch.esp
45  WACCF_BashedPatchLvlListFix.esp
46  Bashed Patch, 0.esp
[/xml][/spoiler]



Bain Packages:
[spoiler][xml]
++ 000 - Unofficial Skyrim Special Edition Patch-266-4-1-9-1560474486.7z (518A14C4) (Installed)
++ 001 - skse64_2_00_16.7z (941BEE21) (Installed)
++ 002 - HavokFix64-18160-2-7-1552913852.zip (07D67322) (Installed)
++ 003 - FNIS Behavior SE 7_5-3038-7-5-1566570336.7z (BEDB809A) (Installed)
++ 004 - FNIS Creature Pack SE 7.0-3038-7-0.7z (C113CCFF) (Installed)
++ 005 - FNIS Spells SE 7.0-3038-7-0.zip (C6316D9A) (Installed)
++ 006 - Caliente's Beautiful Bodies Enhancer - v1.5.3-198-1-5-3-1546894253.7z (27BB0497) (Installed)
++ 007 - CBBE Normal Map Options - v1.4-198-1-4.7z (00658BBB) (Installed)
++ 008 - BodySlide and Outfit Studio - v4.7.4-201-4-7-4-1561250451.7z (4FD9C8D0) (Installed)
++ 009 - RaceMenu Special Edition v0-4-3-19080-0-4-3-1566517443.7z (53ACAA3A) (Installed)
-- 010 - DLL Plugin Loader v1-10546-1.zip (E6D6B414) (Not Installed)
++ 011 - Realistic Ragdolls and Force SE-1439-1-92.zip (3FA0C45C) (Installed)
++ 012 - SkyUI_5_2_SE-12604-5-2SE.7z (C9C79804) (Installed)
++ 013 - PapyrusUtil SE - Scripting Utility Functions-13048-3-7b-1560796241.zip (86EAE331) (Installed)
++ 014 - Papyrus's GameExtension Runtime 1.5.73-13202-1-01-1-5-73-1556963449.zip (8273B07A) (Installed)
++ 015 - Papyrus Extender - SSE 1.5.80 - v1.6-22854-1-6-1563387116.7z (861B2FA5) (Installed)
++ 016 - FileAccess Interface for Skyrim SE Scripts - FISSES-13956-1-3-6-1552757259.zip (932C7FB2) (Installed)
++ 017 - SSE Engine Fixes - Part 1 - 4.6.1 for 1.5.80-17230-4-6-1-1560592054.7z (D28BA8BA) (Installed)
++ 018 - XP32 Maximum Skeleton Special Extended-1988-4-51.7z (D2B4671B) (Installed)
-- 019 - Shae Whisper-Strike SSE - Nord Archer Follower-19366-1-00SE.7z (12D2CDEE) (Not Installed)
++ 020 - cbp.7z-13207-0-25-1560621710.7z (2B547C79) (Installed)
++ 021 - CBPConfigNMM-16486-2.7z (EF7AEEA9) (Installed)
++ 022 - CBP Config Immersive 16380-4.7z (BBD56AE1) (Installed)
++ 023 - CBPC - CBBE_Slim.7z (83F7171A) (Installed)
++ 024 - CBP SINful-24313-4-0-0-1566667800.7z (E4CB5D4E) (Installed)
++ 025 - Cutting Room Floor-276-3-1-5-1563940008.7z (34D2297E) (Installed)
++ 026 - SSE Fixes v3 - 1.5.73-10547-3-1-5-73-1552729022.7z (98AA63E8) (Installed)
++ 027 - Fuz Ro D'oh-15109-1-6-1560767398.7z (B4D2BD17) (Installed)
-- 028 - Cursors Galore Negative SE-20212-1-0.7z (6E7817E4) (Not Installed)
++ 029 - A Quality World Map - Classic with All Roads-5804-8-4CA.zip (1201F642) (Installed)
++ 030 - Simply Optimized Texture for SSE-3858-1-0.zip (3655BAF3) (Installed)
++ 031 - PrivateProfileRedirector SE 0.4-18860-0-4-1561654217.zip (A873FB9B) (Installed)
-- 032 - Performance Textures 1024x1024 v6-21166-v6-1543559248.7z (3D959BE9) (Not Installed)
++ 033 - A Quality World Map - Clear Map Skies-5804-9-0-1CMS.zip (B2C7483B) (Installed)
++ 034 - SMIM SE 2-08-659-2-08.7z (6ED95313) (Installed)
++ 035 - 360 Compass  - v1-14324-1-0.rar (F6E1FC08) (Installed)
++ 036 - Lootable Crates SE-16994-2-0-2.7z (41309841) (Installed)
++ 037 - Named Quicksaves - RealAntithesis Changes with UILib - by RealAntithesis.7z (1F70ECF8) (Installed)
++ 038 - Stay At System Page-19832-1-2-3-1560495099.zip (6C8608FB) (Installed)
++ 039 - Nicola Avenicci SE 0.3-6862-0-3.7z (7FE1D309) (Installed)
++ 040 - Tish SSE - CBBE-4856-1-0.rar (7507707A) (Installed)
++ 041 - Mae CBBE - SSE Conversion-7485-1-0.rar (7554DCBD) (Installed)
++ 042 - Jette Crag-Jumper 1.7-2732-1-8.rar (81491A2A) (Installed)
++ 043 - Extensible Follower Framework v4-0-5-7003-4-0-5.7z (91B4BECE) (Installed)
++ 044 - Left Hand Rings Modified SE-3240-4-5-1542372710.7z (04151FDA) (Installed)
++ 045 - Skyrim SE - Project Optimization - ESM VERSION 1.3-14084-1-3.zip (45374C76) (Installed)
-- 046 - BethINI Standalone Version-4875-3-2-1558876854.zip (5DAA375C) (Not Installed)
++ 047 - Ordinator v926-1137-9-26SSE.zip (6D807D50) (Installed)
++ 048 - A Matter Of Time_v3_0_0_alpha_4-12937-v3-0-0a4.7z (1A6F40B4) (Installed)
++ 049 - Follower Trap Safety-2755-1-3.zip (C8B3F556) (Installed)
++ 050 - Weapons Armor Clothing and Clutter Fixes-18994-1-0-1.7z (E252BA4C) (Installed)
++ 051 - RUSTIC CLOTHING - Special Edition - 2K-1K-4703-2-0.7z (D46A8A34) (Installed)
++ 052 - Armor and Clothing Extension-19002-1-0.7z (932AEBFC) (Installed)
++ 053 - WACCF - CBBE Patch-19176-1-0-1.7z (869DC503) (Installed)
++ 054 - Rustic Armor and Weapons SE - 2K-19666-3-0-1539568066.7z (CD891AAB) (Installed)
++ 055 - Consistency Patch - WACCF and ACE (Merged)-28059-2-1-1565249780.7z (B3EC24A2) (Installed)
++ 056 - Mod Configuration Menu Patch - ACE-28059-1-0-1565199827.7z (07B9A09A) (Installed)
++ 057 - Artful Dodger - Dynamic Pickpocket Cap-17273-1-11.rar (D29BCBAC) (Installed)
++ 058 - Unread Books Glow SSE FOMOD-20679-2-21-1557563525.7z (DA21312E) (Installed)
++ 059 - Pick-Up Books Simple.esp-2453-1-8a-1559517090.zip (20B1F155) (Installed)
++ 060 - Book Covers Skyrim SE - Desaturated.7z-901-4-2.7z (70AA2316) (Installed)
++ 061 - Ordinator __ WACCF-19518-1-0-1547387875.zip (7D358A15) (Installed)
++ 062 - QUASIPC - Unified Patch Compendium v2_5_0-18369-2-5-0-1557483615.7z (6F20ABC7) (Installed)
++ 063 - Relighting Skyrim SSE v1.0.6-8586-1-0-6-1543789763.7z (431CF3C5) (Installed)
++ 064 - Remove 3D Map Camera Limits SE-7012-1-1SE.zip (09F07E51) (Installed)
++ 065 - ACE_MCM_Fix-23471-1-0-1550316439.7z (DE971309) (Installed)
++ 066 - Player Headtracking SE-22210-4-0-1553127660.rar (926A7BF4) (Installed)
++ 067 - Player Headtracking 3_3 Patch-61639-1-2.rar (E51E538C) (Installed)
++ 068 - Eastmarch Ore Veins-6067-1-0.zip (6646A96E) (Installed)
++ 069 - OresFirewoodsEsl1.1.4-23121-1-1-4-1550900545.rar (075D6753) (Installed)
-- 070 - Invisible Headgears-20087-1-1-1540679338.7z (3C294348) (Not Installed)
++ 071 - Invisible Headgears-WACFF+ACE - espfe-20087-1-1-1540679338.7z (ECCC35B6) (Installed)
++ 072 - Thane Weapons Reborn SE-8387-4-2.7z (4070591C) (Installed)
++ 073 - Thane Weapons Reborn - 1k Textures-8387-4-0.7z (D4925B15) (Installed)
++ 074 - Better Dialogue Controls v1_2-1429-1-2.zip (25A02B05) (Installed)
++ 075 - Better MessageBox Controls v1_2-1428-1-2.zip (A52D4AF0) (Installed)
++ 076 - Realistic Conversations w Natural Social  Behaviors-1717-2-0.7z (B8DF8C55) (Installed)
++ 077 - Book Covers Skyrim - Immersive Horses Patch-17700-1-0.zip (4355B88B) (Installed)
++ 078 - Immersive Horses-13402-3-0.7z (7C958AF7) (Installed)
++ 079 - Immersive Horses 2K Horse Textures-13402-1-0.7z (7E4FDCC1) (Installed)
++ 080 - Lilly - Female Courier Replacer (BSA files)-4595-1-1.7z (670143A4) (Installed)
++ 081 - Lilly EVO -19842-5-1-1563995682.7z (24EED746) (Installed)
++ 082 - I'm Walkin' Here-27742-1-0-0-1564216874.zip (4245DA98) (Installed)
-- 083 - Don't Push Me While I'm Talking - SSE Plugin-6444-1-1.zip (655D2DD0) (Not Installed)
++ 084 - No NPC Greetings SE - FOMOD Installer (deprecated-1044-2-0a.rar (A4589526) (Installed)
++ 085 - Move it Dammit SE - Manual - v1.1 (Wrye Bash)-752-1-1-0b.7z (2AAE3F70) (Installed)
++ 086 - No More Standing Too Close SSE-4784-1-0.rar (4D6E08FC) (Installed)
++ 087 - No Killmoves - No Killcams - No Killbites SE-13395-1-1.7z (172B3071) (Installed)
++ 088 - To Your Face SE-24720-1-0g-1560872101.zip (E4C0F634) (Installed)
++ 089 - People Are Strangers-1658-1-0.7z (416B5E84) (Installed)
++ 090 - People Are Strang - Race option - Updates esp File -1658-.7z (348DC26B) (Installed)
++ 091 - Don't Talk With Your Mouth Full-17715-1-1.7z (D9B57460) (Installed)
++ 092 - Shut Up Brynjolf-22017-1-0-1545305061.7z (D8B3CEB2) (Installed)
++ 093 - Charge Dialogue Fix-17716-1-1.7z (89AE86F3) (Installed)
++ 094 - Run For Your Lives-2272-4-0.7z (8DF189DE) (Installed)
++ 095 - NPC Run⁄Walk My Pace-2482-1-00.7z (83317210) (Installed)
++ 096 - Universal Fog Remover - SKSE64_Elys_1_0_2_0-19917-1-0-2-0-1562767179.7z (D43EB44B) (Installed)
097 - ==Last==
[/xml]
```

**Comment by CptMcSplody** [comment]
@pendragg0n Closing this issue as it seems to be resolved.
